### ISSUE TITLE
Update Natural Language Understanding

### ISSUE THREAD

**Issue by Ashxvi** [watson-developer-cloud/swift-sdk#600]
Hi there,
I'm migrating from Alchemy Language API, I want to get the Label instead of the Score of a Sentiment in the new NLU API like this : 
```
naturalLanguageUnderstanding.analyzeContent(withParameters: parameters, failure: failure) {
            results in
            print (results.sentiment?.document?.label)
        }
```
Thanks in advance.

**Comment by schen22** [comment]
Hi @Ashxvi - thanks for letting us know! We'll look into adding the feature (and other missing features we currently have within NLU) , though we probably will not be able to get to them until another two weeks at the soonest. Feel free to open a PR to begin adding support for it if you need it sooner otherwise.

**Comment by Ashxvi** [comment]
Thanks @sche22, I just had an issue while trying to open a PR, I will try to add this feature..

**Comment by glennrfisher** [comment]
Latest version of the Natural Language Understanding service was generated and merged with #684. We'll have an official release in a few days.

**Comment by glennrfisher** [comment]
The latest version of the Natural Language Understanding service is included with release v0.19.0 of the Swift SDK. Hope that helps!
